### ISSUE TITLE
Unusable user_input.yaml

### ISSUE THREAD

**Issue by Maniarr** [awslabs/aws-control-tower-customizations#17]
Hello,

When i use `user_input.yaml` in add-on to templatize some files, values are not correctly replaced by the `find_replace.py` script.

Example, for the variable named `key` with value `value_of_key_variable`.

In template, the string `{{ key }}` will be replaced by `&#34;value_of_key_variable&#34;` instead of `"value_of_key_variable"`.

A small modification of the script introduce this error in v1.1.0.

v1.0.0:
```
# bin/codebuild_scripts/find_replace.py:33
...
        j2env = jinja2.Environment(loader=j2loader)
...
```

v1.1.0:
```
# bin/codebuild_scripts/find_replace.py:35
...
        j2env = Environment(loader=j2loader, autoescape=True)  # Compliant
...
```

Can you reverse it ?

**Comment by groverlalit** [comment]
Thanks @Maniarr for raising this issue. We have added this to our backlog and plan to publish the requested change in the next release.

**Comment by rakshb** [comment]
Hello @Maniarr . We have fixed this issue in CFCT V1.2
